agent/roles/manifests/vars/main.yml

```diff
@@ -5,6 +5,7 @@ agent_nodes_ipsv6: "{{ lookup('env', 'AGENT_NODES_IPSV6_STR') }}"
 agent_nodes_hostnames: "{{ lookup('env', 'AGENT_NODES_HOSTNAMES_STR') }}"
 agent_use_ztp_manifests: "{{ lookup('env', 'AGENT_USE_ZTP_MANIFESTS') }}"
 agent_test_cases: "{{ lookup('env', 'AGENT_TEST_CASES') }}"
+agent_use_appliance_manifest: "{{ lookup('env', 'AGENT_USE_APPLIANCE_MANIFEST') }}"
```

Comment:
Can `AGENT_USE_APPLIANCE_MODEL` be used here instead of adding an additional `AGENT_USE_APPLIANCE_MANIFEST` (or maybe that was the intent and its just a typo?)

Reply:
Actually just wanted to be explicit here, for keeping the convention of other manifests (e.g. AGENT_USE_ZTP_MANIFESTS). We can indeed use AGENT_USE_APPLIANCE_MANIFEST if you think it'd be clearer(?)